contributing/index.md

```diff
@@ -10,7 +10,6 @@ videos
 ../about/contributing_to_pymc/index
 ../about/contributing_to_documentation/index
 donate
-../2022-07_sprint/sprint_parties/media_kit
```

Comment:
I also noticed this was here, so even though the path was sprint specific, it was listed inside the contributing section which was confusing. It is now listed in the "about" of the specific sprint.